docs/user-guide/ambassador-pro-install.md

```diff
@@ -20,7 +20,7 @@ kubectl create secret docker-registry ambassador-pro-registry-credentials --dock
 Ambassador Pro plugins run externally to Ambassador. You will need to download the Ambassador Pro Authentication service deployment in order to change the default values and integrate it with your Auth0 application. Download Ambassador Pro Authentication from https://www.getambassador.io/yaml/ambassador/pro/ambassador-pro-auth.yaml or with:
 
 ```
-wget "https://www.getambassador.io/yaml/ambassador/pro/ambassador-pro-auth.yaml"
+curl https://www.getambassador.io/yaml/ambassador/pro/ambassador-pro-auth.yaml -o ambassador-pro-auth.yaml
```

Comment:
```suggestion
curl -O https://www.getambassador.io/yaml/ambassador/pro/ambassador-pro-auth.yaml
```

nit: if `-O` is used, then the original filename is kept intact and we don't need to specify the filename at all